src/common/components/search/SearchBar.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { FC, useState, useRef, useEffect } from 'react';
```

Comment:
@kristianeboe should comment, but I think under the proposed new code organization, all the stuff you have under the `/search` folder (and potentially `Search.tsx`) would go in one big file, since all of those dependencies are to be used only by `SearchBar` and thus dont need exposure outside of that component.